plugins/nominal-connection-checker/src/index.js

```diff
@@ -216,6 +216,7 @@ export class NominalConnectionChecker extends Blockly.ConnectionChecker {
    */
   isGeneric_(connection) {
     const check = this.getCheck_(connection);
+    // If you ever update this, be sure to update hierarchy_validation.js too.
```

Comment:
It would be better to have these both call the same function. You could make it a static function on GenericMap, or else move it out into a ehlper some other way.